src/chainparams.cpp

```diff
@@ -39,7 +39,9 @@ class CMainParams : public CChainParams {
         // TODO generate harder genesis block
         //consensus.powLimit = uint256S("00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
         consensus.powLimit = uint256S("7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
-        consensus.nPowTargetTimespan = 14 * 24 * 60 * 60; // two weeks
+        consensus.nPowAveragingWindow = 17;
+        consensus.nPowMaxAdjustDown = 16; // 16% adjustment down
+        consensus.nPowMaxAdjustUp = 8; // 8% adjustment up
```

Comment:
There shouldn't be a default value of 0; being undefined in testnet and regtest, they should take on these same parameters.